agent/fw_drupal8.c

```diff
@@ -605,6 +605,89 @@ NR_PHP_WRAPPER(nr_drupal94_invoke_all_with_clean) {
 NR_PHP_WRAPPER_END
 #endif  // OAPI
 
+/*
+ * Purpose: Instrument Drupal Attribute Hooks for Drupal 11.1+
+ *
+ * Params: 1. A zval pointer to the moduleHandler instance in use by Drupal.
+ *
+ * Return: bool
+ *
+ */
+static bool nr_drupal_hook_attribute_instrument(zval* module_handler) {
+  zval* hook_implementation_map = NULL;
+
+  nr_php_string_hash_key_t* hook_key = NULL;
+  zval* hook_val = NULL;
+  nr_php_string_hash_key_t* class_key = NULL;
+  zval* class_val = NULL;
+  nr_php_string_hash_key_t* method_key = NULL;
+  zval* module_val = NULL;
+
+  char* hookpath = NULL;
+
+  hook_implementation_map = nr_php_get_zval_object_property(
+      module_handler, "hookImplementationsMap");
+
+  if (NULL == hook_implementation_map) {
+    nrl_warning(NRL_FRAMEWORK, "NULL hookImplementationsMap object property");
+    return false;
+  }
+
+  if (!nr_php_is_zval_valid_array(hook_implementation_map)) {
```

Comment:
`nr_php_is_zval_valid_array` will already check `hook_implementation_map` for NULL ; it doesn't need the extra check [right above](https://github.com/newrelic/newrelic-php-agent/pull/1030/files#diff-6c636f6e3733aae919f4c4cf66ec5226065346587de8cbaf3918f483f9d0f523R631).

Reply:
removed 2d515f1623b5ce74898dc2679b6ca4f0d16312af